build.gradle

```diff
@@ -9,10 +9,10 @@ buildscript {
 
     repositories {
         google()
-        jcenter()
+        mavenCentral()
```

Comment:
This one is better now?

Reply:
jcenter is no more

Reply:
![image](https://user-images.githubusercontent.com/63658871/204589969-2587afe5-2d7e-4abc-86b9-525967e8554f.png)